internal/go/skycfg/yaml_test.go

```diff
@@ -121,6 +121,31 @@ func TestYamlToSky(t *testing.T) {
 			key:  "mapKey",
 			want: `{"subkey": "val"}`,
 		},
+		{
+			name: "key mapped to Uint",
+			key:  "uintKey",
+			want: `12345678901234567890`,
```

Comment:
I recommend testing specific special boundary values such as `math.MaxInt32` ± 1. It might also be good to have a test for parsing `math.MaxInt64+1`, which will hopefully fail with some reasonable error message.